src/test/java/lotto/domain/LottoNumberTest.java

```diff
@@ -0,0 +1,43 @@
+package lotto.domain;
+
+import static lotto.domain.LottoNumber.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class LottoNumberTest {
+
+    @DisplayName("로또 번호가 1에서 45 사이라면 정상적으로 객체가 생성된다.")
+    @ParameterizedTest
+    @ValueSource(ints = {LOTTO_RANGE_MINIMUM, LOTTO_RANGE_MAXIMUM})
+    void test_LottoNumber(int number) {
+        LottoNumber lottoNumber = LottoNumber.of(number);
+
+        assertThat(lottoNumber).isNotNull();
+    }
+
+    @DisplayName("로또 번호 가 1에서 45 사이가 아니라면 예외를 발생시킨다.")
+    @ParameterizedTest
+    @ValueSource(ints = {LOTTO_RANGE_MINIMUM - 1, LOTTO_RANGE_MAXIMUM + 1})
+    void testLottoNumber_rangeException(int number) {
+        assertThatThrownBy(() -> LottoNumber.of(number))
+                .isInstanceOf(IllegalArgumentException.class);
+    }
+
+    @DisplayName("로또 번호의 비교 결과를 검증한다.")
+    @Test
+    void test_compareTo() {
+        LottoNumber number1 = of(1);
+        LottoNumber number2 = of(1);
+        LottoNumber number3 = of(2);
+
+        assertThat(number1.compareTo(number3)).isNegative();
```

Comment:
```suggestion
        assertThat(number1).isLessThan(number2);
```
`compareTo` 메서드를 검증하는 목적을 달성하고 있지만, 다르게 접근해도 좋아보여요.
`compareTo`를 재정의한 이유가 Comparable의 구현체로 만들기 위해 필요한 요소였잖아요.
때문에 Comparable의 구현체로서 원하는 대로 동작하는지 검증하는 것이 좋지 않을까요?
실제로 sort() 과정에서 비교 행위가 사용되기도 하고, 이를 위해서는 객체 간의 대소 비교가 가능해야 하니까
이를 테스트하기 위해 `assertThat(number1).isLessThan(number2);` 검증을 하면 좋겠어요.